Dockerfile

```
@@ -0,0 +1,138 @@
+FROM ubuntu:xenial as base
+
+# Warning: This file is experimental.
+
+# Install system requirements
+RUN apt update && \
+    # Global requirements
+    DEBIAN_FRONTEND=noninteractive apt install -y \
+    build-essential \
+    curl \
+    # If we don't need gcc, we should remove it.
+    g++ \
+    gcc \
+    git \
+    git-core \
+    language-pack-en \
+    libfreetype6-dev \
+    libmysqlclient-dev \
+    libssl-dev \
+    libxml2-dev \
+    libxmlsec1-dev \
+    libxslt1-dev \
+    software-properties-common \
+    swig \
+    # openedx requirements
+    gettext \
+    gfortran \
+    graphviz \
+    libffi-dev \
+    libfreetype6-dev \
+    libgeos-dev \
+    libgraphviz-dev \
+    libjpeg8-dev \
+    liblapack-dev \
+    libpng-dev \
+    libsqlite3-dev \
+    libxml2-dev \
+    libxmlsec1-dev \
+    libxslt1-dev \
+    ntp \
+    pkg-config \
+    python3-dev \
+    python3-pip \
+    python3.5 \
+    -qy && rm -rf /var/lib/apt/lists/*
```

```suggestion
    rm -rf /var/lib/apt/lists/*
```
* why use quiet?
* we already pass in `--yes`